third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -28,6 +28,11 @@ struct TestConfig {
     runner: fn(&Path) -> datatest_stable::Result<()>,
     experiments: &'static [(&'static str, bool)],
     language_version: LanguageVersion,
+    /// Path substrings for tests to include. If empty, all tests are included.
+    include: &'static [&'static str],
+    /// Path substrings for tests to exclude (applied after the include filter).
```

Comment:
`(applied after the include filter)` is an operational detail.  The results would be the same either way.

Reply:
I find the comment useful. It is in parenthesis anyway, so can be ignored if reader does not care.